examples/API_v2/get_users_mentions.py

```diff
@@ -0,0 +1,19 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace user ID
+id = 2244994945
+
+# By default the Tweet ID and Tweet text will be returned. 
+tweets = client.get_users_mentions(id)
+
+# Print the Tweet text
+for tweet in tweets.data:
+    print(tweet.text)
```

Comment:
I'm still of the opinion this isn't a very useful output, as the text of different Tweets, each with an indeterminate amount of newlines within them, will all be printed out together, making it very difficult to distinguish them.